### ISSUE TITLE
Strange tray icon image

### ISSUE THREAD

**Issue by pahofmann** [IsmaelMartinez/teams-for-linux#35]
I'm running it on a KDE machine and the tray icon looks off. Its a purple box with a white line on the right:
![image](https://user-images.githubusercontent.com/5699104/50107869-15466280-0234-11e9-9eed-7b2ace6d05a4.png)

**Comment by IsmaelMartinez** [comment]
Hi @pahofmann 

Googling a bit I see other people having the same issue with kde.

Solution seems to be in here (on the thread):
https://github.com/vector-im/riot-web/issues/6767#issuecomment-424535414

Hope helps,

**Comment by g4borg** [comment]
i came onto this aswell, after reading through https://github.com/electron/electron/issues/12791 i wanted to add, a quick in-place fix is to modify the application entry in KDE to run teams with a custom export variable prepended, so in layman's terms:

`XDG_CURRENT_DESKTOP=Unity "/opt/teams-for-linux/teams" %U`

instead of

`"/opt/teams-for-linux/teams" %U`

just in case someone else encounters this and wants a quick fix.
